azext_iot/central/command_map.py

```diff
@@ -143,6 +143,19 @@ def load_central_commands(self, _):
         cmd_group.command("compute-device-key", "compute_device_key")
         cmd_group.command("manual-failover", "run_manual_failover")
         cmd_group.command("manual-failback", "run_manual_failback")
+        cmd_group.command("show-attestation", "get_attestation")
```

Comment:
You should put some of these commands into sub groups - like iot central device attestation. Helps with making the iot central device not as cluttered

Reply:
Sure. I'll create some new command groups.